lib/perl/Genome/Config/AnalysisProject/Command/Create.pm

```diff
@@ -125,12 +125,25 @@ sub _get_menu_items {
     } elsif ($self->no_config()) {
         return [];
     } else {
+
+        my $add_config = $self->_ask_user_question(
+            'Would you like to add configuration from the preset config menu items?',
+            60,
```

Comment:
what does the 60 here do?

Reply:
It's the "timeout".  If you don't answer in 60 seconds, it `exit`s.  (That's actually the default in `_ask_user_question`, but made explicit here.)  Can also use `0` for unlimited time.

Reply:
got it! 👍